src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -1,56 +1,50 @@
 import React, { useState } from 'react';
 import { Card } from 'antd';
+import { ModuleView } from '../index';
 import styled from 'styled-components';
 
+//Component Styles
 const CourseCard = styled(Card)`
   margin-bottom: 4%;
 `;
 
-const courseData = [
-  {
-    courseName: 'Python Basics',
-  },
-  {
-    courseName: 'Problem Solving',
-  },
-  {
-    courseName: 'Time and Space Complexity',
-  },
-];
+const Description = styled.div`
+  margin-bottom: 2%;
+`;
 
 const CourseViewContainer = props => {
   const [isExpanded, setIsExpanded] = useState(false);
+
+  //Props passed from ProgramView Component
```

Comment:
This is a wildly helpful comment and I was really pumped to see it! (A lot of your comments in this PR are flame.) But this is a smart pattern, I think I'll start following suit if I am using prop drilling (which is rare, I admit).